src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java

```diff
@@ -123,6 +204,10 @@ public static JsonObject flagToJson(final CheckFlag flag,
         return flagJson;
     }
 
+    private static void accept(final JsonObject jsonObject)
```

Comment:
Any reason for this change?

Reply:
Checkstyle was throwing error for unnecessary parenthesis!! I wasn't sure why I did not get that error before but as a quick turnaround I added this to fix the checkstyle error.